### PR TITLE
Add Report Accident Modal

### LINKED ISSUE #7: Create Incident Capture Page.
**Develop the page for capturing incidents.**

This page page would be a pop up modal on the home page to capture a new incident.
It should contain following fields: 

- Title (max char len=50)

- Description: (max char len=200)

- location: A google map location checker to get the latitude and longitude.

- Reported At: A datetime field.

- Incident type: A dropdown to select incident type.

A json object should be sent to the backend in the format shown on the screenshot. Don't worry about the status field, it's handled at the backend.

Screenshot 2020-05-07 at 6 15 21 PM

### PR DESCRIPTION
**Pull request by fisayo-rm**
## Description
Adds the report accident button which shows the report form in a modal. The button should be used on the landing page.

Fixes #7

### CODE REVIEW

File: src/components/ReportModal.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    e.preventDefault();
+    const obj = {
+      title: report.title,
+      description: report.description,
+      location: {
+        map_name: address,
+        known_name: report.location,
+        location_lat: coordinates.lat,
+        location_lng: coordinates.lng
+        // viewport_ne_lat: 40.7137011802915,
```
Comment (Techiefrankie) [inline src/components/ReportModal.js]: Please remove unused codes in comments

**Review by Techiefrankie — CHANGES_REQUESTED** [review]
Everything looks good to me. Please can you remove all commented codes not used to make code more readable and concise, and also to meet the standards we have adopted.

File: src/components/ReportModal.js (hunk 1 above)
Comment (fisayo-rm) [reply src/components/ReportModal.js]: The comments have been removed.

**Comment by Techiefrankie** [comment]
@fisayo-DMG please could do a pull from develop branch to update your local with the changes that @haffs0 made that was merged into the develop? This is to avoid a merge conflict. Thank you.

**Comment by fisayo-rm** [comment]
I did that already.

On 2 Jun 2020 10:44, "Franklyn Ogbonna" <notifications@github.com> wrote:

@fisayo-DMG <https://github.com/fisayo-DMG> please could do a pull from
develop branch to update your local with the changes that @haffs0
<https://github.com/haffs0> made that was merged into the develop? This is
to avoid a merge conflict. Thank you.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/BuildForSDG/Team-079-Frontend/pull/9#issuecomment-637424659>,
or unsubscribe

**Comment by Techiefrankie** [comment]
Great then. Thanks.

**Review by Techiefrankie — APPROVED** [review]

**Pull request merged**